src/AppBundle/Command/DeleteUserCommand.php

```diff
@@ -0,0 +1,149 @@
+            ->setHelp(<<<HELP
+The <info>%command.name%</info> command deletes users from the database:
+
+  <info>php %command.full_name%</info> <comment>username</comment>
+
+If you omit the argument, the command will ask you to
+provide the missing value:
+
+  <info>php %command.full_name%</info>
+HELP
+            );
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->entityManager = $this->getContainer()->get('doctrine')->getManager();
+    }
+
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        if (null !== $input->getArgument('username')) {
+            return;
+        }
+
+        $output->writeln('');
+        $output->writeln('Delete User Command Interactive Wizard');
+        $output->writeln('-----------------------------------');
+
+        $output->writeln(array(
+            '',
+            'If you prefer to not use this interactive wizard, provide the',
+            'arguments required by this command as follows:',
+            '',
+            ' $ php app/console app:delete-user username',
+            '',
+        ));
+
+        $output->writeln(array(
+            '',
+            'Now we\'ll ask you for the value of all the missing command arguments.',
+            '',
+        ));
+
+        $helper = $this->getHelper('question');
+
+        $question = new Question(' > <info>Username</info>: ');
+        $question->setValidator(array($this, 'usernameValidator'));
+        $question->setMaxAttempts(self::MAX_ATTEMPTS);
+
+        $username = $helper->ask($input, $output, $question);
+        $input->setArgument('username', $username);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $username = $input->getArgument('username');
+        $this->usernameValidator($username);
+
+        $repository = $this->entityManager->getRepository('AppBundle:User');
+        /** @var User $user */
+        $user = $repository->findOneByUsername($username);
+
+        if (null === $user) {
+            throw new \RuntimeException(sprintf('User with username "%s" not found.', $username));
+        }
+
+        // After an entity has been removed its in-memory state is the same
+        // as before the removal, except for generated identifiers.
+        // See http://docs.doctrine-project.org/en/latest/reference/working-with-objects.html#removing-entities
+        $userId = $user->getId();
+
+        $this->entityManager->remove($user);
+        $this->entityManager->flush();
+
+        $output->writeln('');
+        $output->writeln(sprintf('[OK] User "%s" (ID: %d, email: %s) was successfully deleted.', $user->getUsername(), $userId, $user->getEmail()));
+    }
+
+    /**
+     * This internal method should be private, but it's declared public to
+     * maintain PHP 5.3 compatibility when using it in a callback.
+     *
+     * @internal
+     */
```

Comment:
__[development][code_sniffer]__
* Missing @return tag in function comment

<!-- Comment by [SideCI](https://sideci.com) -->